modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/integration/AbstractBasicIntegrationTest.java

```diff
@@ -82,7 +84,7 @@ protected QueryChecker assertQuery(String qry) {
     }
 
     /** */
-    protected IgniteCache<Integer, Employer> createAndPopulateTable() {
+    protected IgniteCache<Integer, Employer> createAndPopulateTable() throws InterruptedException {
```

Comment:
Actually it's not throwing any InterruptedException. Please fix it and any test using this method